### ISSUE TITLE
Issue with hanging request

### ISSUE THREAD

**Issue by TomYeoman** [3846masa/axios-cookiejar-support#196]
Hi thanks for creating this library it's just what i'm looking for!

I'm attempting to use this solution on an API which returns a 302 initially with a set-cookie JSESSIONID which is then needed on subsequent requests.

My issue is that i'm unfortunately getting timed out on my requests after I add the code

```
import axios from 'axios';
import axiosCookieJarSupport from 'axios-cookiejar-support';
import { CookieJar } from 'tough-cookie';

const cookieJar = new CookieJar()

axiosCookieJarSupport(axios)
const result = await axios.post(`${baseUrl}${endpoints.insertOrder}`, form, {
	headers: {
		'content-type': `multipart/form-data; boundary=${form.getBoundary()}`,
	},
	jar: cookieJar,
	withCredentials: true
});
```

I've tried the following with the request library and it works fine - am I missing something?

```
const res = await requestPromise({
	method: 'POST',
	followAllRedirects: true,
	uri: `${baseUrl}${endpoints.insertOrder}`,
	form: {
		...order,
		products: JSON.stringify(order.products),
	},
	headers: {
		'Content-Type': 'multipart/form-data',
	},
	jar: true,
	withCredentials: true,
	resolveWithFullResponse: true,
});
```

**Comment by prfss** [comment]
Having the same issue.
The problem occurs when used with `form-data` and received 302 redirection. Following code eventually throws `socket hang up` error.

```
import axios from "axios";
import axiosCookieJarSupport from "axios-cookiejar-support";
import { CookieJar } from "tough-cookie";
import FormData from "form-data";

const myaxios = axios.create();

const jar = new CookieJar();
axiosCookieJarSupport(myaxios);
myaxios.defaults.jar = jar;

async function main() {
    try {
        const form = new FormData();

        form.append("foo", 123);
        form.append("bar", 456);

        const res = await myaxios.post("http://localhost:3000/redirect", form, {
            headers: form.getHeaders()
        });

        console.log(res.data);
    } catch (e) {
        console.log(e);
    }
}

main();
```

From server side, the first request is observable but the second one (request for redirected page) is not.

axios: 0.21.0
axios-cookiejar-support: 1.0.1

**Comment by prfss** [comment]
I think the problem is caused by 'GET with payload'.

 I added `config.data = undefined;` after line 41 of `response.js` and the redirection is processed successfully.

https://github.com/3846masa/axios-cookiejar-support/blob/31564860a4a750230f297d559e528806a9b45ea0/lib/interceptors/response.mjs#L39-L42

**Comment by 3846masa** [comment]
Sorry for the delay in response.
This issue has been resolved with the release of v2.0.0.

If you think this issue is still unresolved, please reopen it.
